### DISCUSSION TITLE
Support for different out dir in swc jest

### DISCUSSION THREAD

**Discussion by shoham-utila** [swc-project/swc#8388] [Q&A] [answered]
Is there a way to pass `out-dir` to jest when running with swc? 
If not, I think opening an issue with feature request to support `out-dir` through the config so it could be passed through the options in the jest config, which may be relevant for the entire project and not just swc/jest

**Comment by kdy1** [comment, marked as answer]
`out-dir` is an option for CLI

**Comment by shoham-utila** [comment]
@kdy1 It is, I think it should be supported in the config as well, 
my specific use case is for jest which doesn't run with the cli but with the jest cli, but I believe it would be helpful to include it perhaps in the config both to prevent repetition when using the cli (adding out-dir and the path) each run, or when running programmatically.

**Reply by kwonoj** [reply]
if you're using jest's cli and have configured with jest with swc transformer, you wouldn't care about its outdir. If you are using swc's cli as separate transform step you should able to configure outdir with it. I don't think at this point we'd consider specific usecases for you.
